src/xperm_rules.md

```diff
@@ -74,6 +75,26 @@ Conditional Policy Statements
 | ----------------------- | ----------------------- | ----------------------- |
 | No                      | No                      | No                      |
 
+### Extended Permission Evaluation
+
+Extended permission rules are evaluated as follows:
+
+* If no extended permissions are defined, only the resource-level policy is
+  considered.
+
+* If an extended permission rule is defined, the policy is first evaluated
+  according to the high-level resource policy. For example:
+
+  * If an *allowxperm* rule is defined, extended permissions will only be
+    granted if *allow* is granted to the resource. 
+
+  * If an *auditallowxperm* rule is defined, extended audit permissions will only 
+    be granted if *auditallow* is granted to the resource. 
```

Comment:
I think we should avoid the words "granted" and "permissions" in this context when we're not talking about permissions, just auditing.  "extended auditing will only be performed..."

Reply:
Good point; done.